### PR TITLE
Added sqrinv_seq_farm_pipe sample - Issue 313

### PR DESCRIPTION
**Pull request by mdolz**
Added sqrinv_seq example using the farm+pipeline composition.

### CODE REVIEW

File: samples/farm/farm_pipeline/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+add_executable(sqrinv_seq_farm_pipe main.cpp )
+
+target_link_libraries(sqrinv_seq_farm_pipe
+  ${CMAKE_THREAD_LIBS_INIT} 
+  ${TBB_LIBRARIES} 
+  ${Boost_LIBRARIES} )
```
Comment (jdgarciauc3m) [inline samples/farm/farm_pipeline/CMakeLists.txt]: Why boost?

**Review by jdgarciauc3m — CHANGES_REQUESTED** [review]
We should not depend on Boost

**Comment by mdolz** [comment]
Fixed.

**Review by jdgarciauc3m — APPROVED** [review]
OK.

**Comment by jdgarciauc3m** [comment]
This solves issue #313

**Pull request merged**